doc/configuration.md

```diff
@@ -355,6 +406,43 @@ Enables or disables multi assets tables and entries.
 
  * Type: `boolean`
 
+#### Policies
+
+Whitelist for multi asset policies hash. Only set values will be kept in the database, all others will be ignored.
+
+`policies`
+ 
+ * Type: `string`
+
+The string will be validated and needs to omit `\x` from the start of the hash eg:
+
+`\x6c969320597b755454ff3653ad09725d590c570827a129aeb4385526`
+should be entered as:
+`6c969320597b755454ff3653ad09725d590c570827a129aeb4385526`
+
+Be mindfull that whitelisting policies will omit data that does not match whitelist being present from the following tables.
+
++-----------------------+
+| table name            |
+| ----------------------+
+| datum                 |
+| script                |
+| tx_out                |
+| multi_assets          |
++-----------------------+
```

Comment:
Is this all that is omitted or are the whole outputs of the tx omitted as well? Are there queries done on these outputs and how are they affected?